### DISCUSSION TITLE
[Unstable] Create new character is available in PVP modes

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14479] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

You can create a new character in the middle of pvp rounds. Doesn't affect anything if respawning isn't enabled, but if it is, then things break pretty badly - trying to swap teams leads to respawning on a shuttle while being on neither team.
This shouldn't be accessible in pvp.

### Reproduction steps

1. start a pvp round (I used outpost deathmatch)
2. go into your character menu and create a new character on the opposite team
3. observe that you are not on either team and are on a respawn shuttle

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested, tested the fix to be working correctly. Closing discussion.
